formbricks/formbricks
# feat: Make formbricks-js ready for public websites

## What does this PR do?



Fixes # (issue)



## Type of change




## How should this be tested?



- Make sure in-App Surveys are working properly using the Demo App.
- Test both with and without `User Identification`
- Check if Link Surveys are working properly (Displays + Responses)
- Check if survey targeting is working with user identification
- Check if recontact options are working with both activated/deactivated user identification
- Check if action creation and response-person connection work with user identification enabled

## Checklist



### Required


### Appreciated
